components/search/SearchQueryOptions.vue

```diff
@@ -0,0 +1,434 @@
+<template>
+  <b-list-group
+    v-show="query && query.length > 0"
+    :id="elementId"
+    class="auto-suggest-dropdown"
+    data-qa="search suggestions"
+    role="listbox"
+    :aria-label="$t('searchSuggestions')"
+    :aria-hidden="!isActive"
+  >
+    <b-list-group-item
+      v-if="showLoader && isLoading"
+      class="loading"
+    >
+      {{ $t('loadingResults') }}{{ $t('formatting.ellipsis') }}
+    </b-list-group-item>
+
+    <template v-else>
+      <template
+        v-if="pillLabel"
```

Comment:
`pillLabel` doesn't make any sense as a variable name anymore. It should be renamed to 'entityCollectionLabel'

Reply:
yep good point